header/entropy/entropy.hpp

```diff
@@ -0,0 +1,46 @@
+/************************/
+/**     Libraries      **/
+/************************/
+#include <string>
+#include <map>
+#include <cmath>
+/************************/
+/**     Functions      **/
+/************************/
+namespace Entropy
+{
+    /**
+     * @brief Function to check if a string entropy fits between two value
+     * 
+     * @param _string The string we want to calculate the entropy of
+     * @param entropy_min The minimum entropy the string must have
+     * @param entropy_max The maximum entropy the string must have
+     * @return true 
+     * @return false 
+     */
+    bool stringFitEntropy(
+        std::string _string,
```

Comment:
std::string -> const std::string&
& -> by reference : not copying the string -> always use references for large objects
const -> allows to pass inline strings like "test", not only variables. Only if your function doesn't modify the string